requirements.txt

```diff
@@ -1,3 +1,4 @@
 Flask==1.0.2
 sqlalchemy==1.2.15
 isoweek==1.3.3
+requests==2.21.0
```

Comment:
My my, look what's back

Reply:
Cant do anything useful in python without requests
`import requests
print(1+1)`